invenio_oauthclient/contrib/cern.py

```diff
@@ -163,6 +163,15 @@
 REMOTE_APP_RESOURCE_API_URL = 'https://oauthresource.web.cern.ch/api/Me'
 REMOTE_APP_RESOURCE_SCHEMA = 'http://schemas.xmlsoap.org/claims/'
 
+cern_oauth_blueprint = Blueprint('cern_oauth', __name__)
+
+
+@cern_oauth_blueprint.route('/cern/logout')
+def logout():
+    """CERN logout view."""
+    return redirect("https://login.cern.ch/adfs/ls/partiallogout.aspx",
```

Comment:
I think it would be nice to have this url configurable... WDYT?
You can put it in `REMOTE_APP` config maybe.

Reply:
I also think the logout url should be in config variable